packages/mongo-nav/src/helpers/Icons.tsx

```diff
@@ -6,6 +6,146 @@ type ProductIconProps = HTMLElementProps<'svg'> & {
   active: boolean;
 };
 
+export function CloudManagerSmall() {
```

Comment:
Is there a reason we need this in addition to the regular CM logo component? We should just be able to set the flat colors for the fill instead of the gradients as a part of a variant.

Reply:
Ditto for Atlas. It might be nice to determine whether to display the flat colors vs gradients based on a size prop actually!

Reply:
👍